tests/specification_tests/test_client_specs.py

```diff
@@ -29,6 +29,9 @@ def load_specs():
 def iter_spec():
     for spec in load_specs():
         name, state, tests, variant_tests = load_spec(spec)
+        state["features"] = [
+            {"impressionData": False, "stale": False, **feature} for feature in state["features"]
+        ]
```

Comment:
I should be able to remove these lines once https://github.com/Unleash/client-specification/pull/51 has merged

Reply:
Hopefully this won't be necessary if we address the previous comment